components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/constant/APIResourceManagementConstants.java

```diff
@@ -98,6 +99,10 @@ public enum ErrorMessages {
                 " scope.", "Error while checking existence of scope in the database."),
         ERROR_CODE_ERROR_WHILE_CHECKING_API_RESOURCE_EXISTENCE("65011", "Error while checking existence " +
                 "of API resource.", "Error while checking existence of API resource in the database."),
+        ERROR_CODE_ERROR_WHILE_RETRIEVING_API_RESOURCE_PROPERTIES("65013", "Error while retrieving API " +
```

Comment:
why 65012 is missing?

Reply:
This is done intentionally because of this PR. (https://github.com/wso2/carbon-identity-framework/pull/5003)